examples/remotereconnection/src/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const Prism = require('prismjs');
+const Video = require('twilio-video');
+const getSnippet = require('../../util/getsnippet');
+const getRoomCredentials = require('../../util/getroomcredentials');
+const helpers = require('./helpers');
+const remoteReconnectionUpdates = helpers.remoteReconnectionUpdates;
+
+const p1Media = document.getElementById('p1-media');
+const P1simulateReconnection = document.getElementById('p1-simulate-reconnection');
+
+// Update UI to indicate remote side room state changes
+const onRoomStateChange = (participant, newState) => {
+  const oldRoomState = document.querySelector(`#${participant} div.current`)
+  if (oldRoomState) {
+    oldRoomState.classList.remove('current');
+  }
+
+  const newRoomState = document.querySelector(`#${participant} div.${newState}`)
+  newRoomState.classList.add('current');
+}
+
+(async function() {
+  // Load the code snippet.
+  const snippet = await getSnippet('./helpers.js');
+  const pre = document.querySelector('pre.language-javascript');
+  
+  pre.innerHTML = Prism.highlight(snippet, Prism.languages.javascript);
+  
+  // Get the credentials to connect to the Room.
+  const credsP1 = await getRoomCredentials();
+  const credsP2 = await getRoomCredentials();
+  
+  // Create room instance and name for participants to join.
+  const roomP1 = await Video.connect(credsP1.token);
```

Comment:
```suggestion
  const roomP1 = await Video.connect(credsP1.token, { tracks: [] });
```

Reply:
Do we not want the local participant (P1) to show video/audio? Or should I make P2 to not publish tracks?

Reply:
@PikaJoyce Ideally you should be using only one of the Rooms to showcase both LocalParticipant and RemoteParticipant reconnection states (RoomP1 in this example). So, when you close the connection for RoomP2, RoomP1 emits a "participantReconnecting" event. When you close the connection for RoomP1, then RoomP1.localParticipant emits a "reconnecting" event.